modules/tracing/mule-tracer-configuration-impl/src/test/java/org/mule/runtime/tracing/level/impl/config/FileTracingLevelConfigurationTestCase.java

```diff
@@ -228,8 +250,10 @@ private static class TestFileTracingLevelWithWrongOverrideConfiguration extends
 
     public static final String TEST_CONF_FILE_NAME = TRACING_LEVEL_WITH_WRONG_OVERRIDE_CONF;
 
-    public TestFileTracingLevelWithWrongOverrideConfiguration(MuleContext muleContext) {
+    public TestFileTracingLevelWithWrongOverrideConfiguration(MuleContext muleContext,
+                                                              SpanExporterConfiguration spanExporterConfiguration) {
       super(muleContext);
+      this.spanExporterConfiguration = spanExporterConfiguration;
```

Comment:
instead of setting this field as protected, inject through a setter as mentioned above.